integration-tests/tests/home_interface_test.go

```diff
@@ -0,0 +1,146 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+// +build !excludeintegration
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package tests
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/ubuntu-core/snappy/integration-tests/testutils/cli"
+	"github.com/ubuntu-core/snappy/integration-tests/testutils/data"
+
+	"gopkg.in/check.v1"
+)
+
+var _ = check.Suite(&homeInterfaceSuite{
+	interfaceSuite: interfaceSuite{
+		sampleSnap: data.HomeConsumerSnapName,
+		slot:       "home",
+		plug:       "home-consumer"}})
```

Comment:
probably the plug field name is a bit misleading, it is intended to be interpolated in these strings https://github.com/fgimenez/snappy/blob/home-interface-int-test/integration-tests/tests/interfaces_test.go#L34 so that there are lines that read `:home home-consumer` when the plug is connected and `:home -\n...\n- home-consumer:home` when it is disconnected... Any suggestions for picking a better name welcomed :)